graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -530,49 +533,55 @@ class RenameSchemaTypesVisitor(Visitor):
         }
     )
     # Collects naming conflict errors involving types that are not built-in scalar types. If
-    # renaming would result in multiple types being named "Foo", type_name_conflicts will map "Foo"
-    # to a set containing the name of each such type
+    # type_renamings would result in multiple types being named "Foo", type_name_conflicts will map
+    # "Foo" to a set containing the name of each such type
     type_name_conflicts: Dict[str, Set[str]]
-    # Collects naming conflict errors involving built-in scalar types. If renaming would rename a
-    # type named "Foo" to "String", renamed_to_scalar_conflicts will map "Foo" to "String"
-    renamed_to_builtin_scalar_conflicts: Dict[str, str]
+    # Collects naming conflict errors involving built-in scalar types. If type_renamings would
+    # rename a type named "Foo" to "String", type_renamed_to_builtin_scalar_conflicts will map
+    # "Foo" to "String"
+    type_renamed_to_builtin_scalar_conflicts: Dict[str, str]
     # reverse_name_map maps renamed type name to original type name, containing all non-suppressed
     # non-scalar types, including those that were unchanged. Must contain unchanged names to prevent
-    # renaming conflicts and raise SchemaRenameNameConflictError when they arise
+    # type renaming conflicts and raise SchemaRenameNameConflictError when they arise
     reverse_name_map: Dict[str, str]
-    # Collects invalid type names in renamings. If renaming would rename a type named "Foo" to a
-    # string that is not a valid, unreserved GraphQL type name (see definition in the
-    # type_name_is_valid function in utils), invalid_type_names will map "Foo" to the invalid type
-    # name.
+    # Collects invalid type names in type_renamings. If type_renamings would rename a type named
+    # "Foo" to a string that is not a valid, unreserved GraphQL type name (valid, unreserved names
+    # consist only of alphanumeric characters and underscores, do not start with a number, and do
+    # not start with two underscores), invalid_type_names will map "Foo" to the invalid type name.
     invalid_type_names: Dict[str, str]
-    # Collects the type names for types that get suppressed. If renaming would suppress a type named
-    # "Foo", renamed_types will contain "Foo".
-    suppressed_types: Set[str]
+    # Collects the type names for types that get suppressed. If type_renamings would suppress a type
+    # named "Foo", suppressed_type_names will contain "Foo".
+    suppressed_type_names: Set[str]
+    # reverse_field_name_map maps type name to a dict, which in turn maps the name of a field in the
```

Comment:
This seems new, and doesn't seem like it refers to any particular attribute here.

For readability and to make errors like this easier to spot, consider using new lines to visually separate the comments from the code above them (the code not related to them).